### PR TITLE
Vp2420 fixes

### PR DESCRIPTION
**Pull request by macpijan**
We still need to take a look at:
- [ ]  https://github.com/Dasharo/dasharo-issues/issues/740 - likely a change in tests
- [ ] Proper frequency calculation and offset as discussed in: https://github.com/Dasharo/dasharo-issues/issues/763

### CODE REVIEW

**Review by filipleple — APPROVED** [review]
LGTM - does fix `Boot Time Measure`

**Comment by filipleple** [comment]
Windows tests fail after 5 mins, probably due to autosleep - which is strange, since I recall disabling it.  Will investigate and fix

File: platform-configs/novacustom-ns50mu.robot [hunk 1]
```diff
@@ -17,306 +17,18 @@ ${IPXE_BOOT_ENTRY}=                                 iPXE Network boot
 ${AUTO_BOOT_TIME_OUT_DEFAULT_VALUE}=                ${EMPTY}
 ${LAPTOP_EC_SERIAL_WORKAROUND}=                     ${TRUE}
 
-# Platform flashing flags
```
Comment (pkubaj) [inline platform-configs/novacustom-ns50mu.robot:20]: Is it necessary to remove 90% of this file?

Comment (macpijan) [reply platform-configs/novacustom-ns50mu.robot]: I see only indentation changes here. I assume this comment was for older patch version.

**Pull request merged**